dev/README_RELEASE_HELM_CHART.md

```diff
@@ -122,65 +122,66 @@ official Apache releases must not include the rcN suffix.
     ```
 
 - Update Helm Chart version in `Chart.yaml`, example: `version: 1.0.0` (without
-  the RC tag). If the default version of Airflow is different from `appVersion` change it.
+  the RC tag).
```

Comment:
Do we not need to take care of it anymore - "If the default version of Airflow is different from `appVersion` change it."

Reply:
It's part of the Airflow release process, and should also be caught in reviews when it gets bumped:
https://github.com/apache/airflow/blob/0afe3d129db2792f58f6fd8cc5e278aba9cf166c/dev/README_RELEASE_AIRFLOW.md?plain=1#L966-L968

Reply:
We can leave this in though, I don't have a strong preference.

Reply:
I will vote for keeping it in case this was missed when releasing Airflow